wgpu-types/src/lib.rs

```diff
@@ -3253,6 +3255,8 @@ impl TextureFormat {
             allowed_usages.contains(TextureUsages::STORAGE_ATOMIC),
         );
 
+        allowed_usages |= TextureUsages::TRANSIENT;
```

Comment:
Hmm I don't think this is actually true - I think it needs to be only the textures which have RENDER_ATTACHMENT.

You can include this in the `attachment` variable.